lib/cadence/workflow/history/event_target.rb

```diff
@@ -37,43 +38,72 @@ class UnexpectedEventType < InternalError; end
           'WorkflowExecution'                      => WORKFLOW_TYPE,
         }.freeze
 
-        attr_reader :id, :type
+        DECISION_TARGET_TYPES = {
+          'Cadence::Workflow::Decision::ScheduleActivity'            => History::EventTarget::ACTIVITY_TYPE,
+          'Cadence::Workflow::Decision::RequestActivityCancellation' => History::EventTarget::CANCEL_ACTIVITY_REQUEST_TYPE,
+          'Cadence::Workflow::Decision::RecordMarker'                => History::EventTarget::MARKER_TYPE,
+          'Cadence::Workflow::Decision::StartTimer'                  => History::EventTarget::TIMER_TYPE,
+          'Cadence::Workflow::Decision::CancelTimer'                 => History::EventTarget::CANCEL_TIMER_REQUEST_TYPE,
+          'Cadence::Workflow::Decision::CompleteWorkflow'            => History::EventTarget::WORKFLOW_TYPE,
+          'Cadence::Workflow::Decision::FailWorkflow'                => History::EventTarget::WORKFLOW_TYPE,
+          'Cadence::Workflow::Decision::StartChildWorkflow'          => History::EventTarget::CHILD_WORKFLOW_TYPE,
```

Comment:
Nit: Can you either remove the `History::EventTarget::` qualification from the `DECISION_TARGET_TYPES` values or fully qualify the `EVENT_TARGET_TYPES` values with `History::EventTarget::`?

Reply:
@DeRauk addressed